addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -98,7 +98,7 @@ $text-muted: mute-color($body-color) !default;
 
 // HR
 $hr-color: $black !default;
-$hr-opacity: 0.1 !default;
+$hr-opacity: 1 !default;
```

Comment:
I would put this in web/bootstrap_overridden_frontend.scss, not the website one (or at least in the portal one, which is kinda the same). In general, anything which affects the frontend should be in that one if it is not related to a website variable (user configuration). That way if the frontend is used without website and then the website is installed, the HR behavior does not change (= we can use those HR the same way in frontend templates whether website is installed or not).

Reply:
I also changed the default color to have the same color as in BS4 now that the "0.1" opacity has been removed